libbeat/dashboards/import_dashboards.go

```diff
@@ -0,0 +1,559 @@
+package main
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/fmtstr"
+	"github.com/elastic/beats/libbeat/outputs/elasticsearch"
+	"github.com/elastic/beats/libbeat/outputs/outil"
+)
+
+type Options struct {
+	KibanaIndex string
+	ES          string
+	Index       string
+	Dir         string
+	File        string
+	Beat        string
+	Url         string
+	User        string
+	Pass        string
+}
+
+type CommandLine struct {
+	flagSet *flag.FlagSet
+	opt     Options
+}
+
+type Importer struct {
+	cl     *CommandLine
+	client *elasticsearch.Client
+}
+
+func ParseCommandLine() (*CommandLine, error) {
+	var cl CommandLine
+
+	cl.flagSet = flag.NewFlagSet("import", flag.ContinueOnError)
+
+	cl.flagSet.Usage = func() {
+		const usage = `
+Usage: ./import_dashboards [options]
+
+Kibana dashboards are stored in a special index in Elasticsearch together with the searches, visualizations, and indexes that they use.
+
+You can import the dashboards, visualizations, searches, and the index pattern for any Beat:
+  1. from a local directory:
+	./import_dashboards -dir etc/kibana
+  2. from a directory of a local zip archive:
+	./import_dashboards -dir metricbeat -file beats-dashboards-1.2.3.zip
```

Comment:
Right, actually I was planning to update the url (from the 3rd point) after we create the archive with the beats dashboards (needs to come in a separate PR) in order to provide a valid url in these examples.